R/pkg/R/functions.R

```diff
@@ -1986,15 +1998,20 @@ setMethod("levenshtein", signature(y = "Column"),
 #' are on the same day of month, or both are the last day of month, time of day will be ignored.
 #' Otherwise, the difference is calculated based on 31 days per month, and rounded to 8 digits.
 #'
+#' @param roundOff an optional parameter to specify if the result is rounded off to 8 digits
 #' @rdname column_datetime_diff_functions
 #' @aliases months_between months_between,Column-method
 #' @note months_between since 1.5.0
 setMethod("months_between", signature(y = "Column"),
-          function(y, x) {
+          function(y, x, roundOff = NULL) {
```

Comment:
Can we add a simple check if `roundOff` is a logical or not?

Reply:
we are not doing that consistently, so I'm ok with if a check or 
`callJStatic("org.apache.spark.sql.functions", "months_between", y@jc, x, as.logical(roundOff))`